README.md

```diff
@@ -39,6 +39,8 @@ the project page of [Vimb][].
 
 - gtk+-3.0
 - webkit2gtk-4.0 >= 2.20.x
+- gst-libav (for media decoding)
+- gst-plugins-good (for nonfree media decoding)
```

Comment:
Strictly speaking both are optional and it would be nice to says that in some ways, maybe adjusting the wording to maybe just one item, e.g.:

```
 - gst-libav, gst-plugins-good (optional, needed for media playback)
```

(I also do not think that gst-plugins-good are useful for "nonfree" media only.)

Reply:
Added 'optional' to description also mentioned that 'gst-plugins-good' might be needed for other things as well.